shared/routes/Repl/index.html

```diff
@@ -0,0 +1,426 @@
+<div class='repl-container'>
+	<div class='left' style='width: {{verticalDividerPos}}%;'>
+		<section class='input'>
+			<div class='controls'>
+				<button class='{{showGenerated ? "" : "active"}}' on:click='set({ showGenerated: false })'>input</button>
+				<button class='{{showGenerated ? "active" : ""}}' on:click='set({ showGenerated: true })'>output</button>
+				<button class='{{saving ? "active" : ""}}' on:click='save()'>{{saving ? 'saving' : 'save'}}</button>
```

Comment:
I associate `submit` with forms, rather than event triggers. You could maybe argue the 'save' button is, but the others aren't, they're effectively tabs. With `submit` you have to prevent the event default — is there any real advantage to `submit`? If it's an a11y thing, maybe there's an `aria` attribute that would be more appropriate?